app/config.js

```diff
@@ -42,7 +45,7 @@ module.exports = {
   redisUrl: process.env.REDIS_URL || 'localhost',
 
   // The port redis is running
-  redisPort: process.env.REDIS_PORT || 6379,
+  redisPort: process.env.REDIS_PORT || 6380,
```

Comment:
oh yeaahhh!!!! makes me wonder how I'm getting it to work for me locally via the cluster

Reply:
I think the cluster one is 6379 and local is 6380 so I will change it back

Reply:
yeah that's right. got confused